__tests__/user/user.mutation.test.js

```diff
@@ -766,3 +763,392 @@ describe('User`s mutation restictions tests', () => {
     expect(result.graphQLErrors[0].message).toEqual('USER_NOT_AUTHORIZED');
   });
 });
+
+describe('Register admin', () => {
+  let superAdminToken;
+  let role = 'admin';
+  let id;
+  let token;
+  let invalidFirstName = 'H';
+  let invalidLastName = 'O';
+  let invalidPassword = 'You';
+  let invalidEmail = 'invalid@com';
+  let adminEmail = adminUser.email;
+  let invalidRole = 'superadmin';
+  let invalidToken = `ayJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2
+    VySWQiOiI1ZjU0ZDY1NDE0NWJiNzM3NzQxYmNmMDMiLCJlbWFpbCI6InN1c
+    GVyYWRtaW5AZ21haWwuY29tIiwiaWF0IjoxNTk5Mzk1NDEyfQ.
+    5z1BRqzxF41xmgKr3nDEDBjrv8TxrkOubAEZ3hEOZcw`;
+  let {
+    email: newAdminEmail,
+    password: newAdminPassword,
+    firstName: newAdminFirstName,
+    lastName: newAdminLastName,
+  } = newAdmin;
+
+  beforeAll(async () => {
+    superAdminToken = await adminLogin(superAdminUser);
+  });
+
+  afterAll(async () => {
+    await client.mutate({
+      mutation: gql`
+        mutation($id: ID!) {
+          deleteUser(id: $id) {
+            firstName
+          }
+        }
+      `,
+      variables: {
+        id,
+      },
+      context: {
+        headers: {
+          token: superAdminToken,
+        },
+      },
+    });
+  });
+
+  test('Should throw an error when use already in-usage email while admin registration', async () => {
```

Comment:
Please, create here different `describe` statements for the different admin operations to clearly separate tests for `registerAdmin`, `completeRegisterAdmin` and `loginAdmin`

Reply:
l did it